drivers/serial/Kconfig

```diff
@@ -125,6 +125,14 @@ config UART_PIPE
 	  data (as contrary to console UART driver) and all aspects of received
 	  protocol data are handled by application provided callback.
 
+config UART_ASYNC_RX
```

Comment:
`UART_ASYNC_RX_HELPER` or `UART_ASYNC_RX_AGG` (aggregator) or `UART_ASYNC_RX_POOL` (receiving pool but will definitely cause confusion with polling)?

Reply:
Changed to `UART_ASYNC_RX_HELPER`. I did not go with renaming files from `uart_async_rx.[ch]` to `uart_async_rx_helper.[ch]`. What do you think? If renaming the file what about API calls? adding `_helper_` part will make names quite long.

Reply:
Hm... how about naming the whole thing as "buffered RX"? So the prefix could be `uart_async_buf_rx_`. I think this is more clear on its purpose than simple "helper".